src/app/migration/src/src/app/modules/core/services/user/user.service.ts

```diff
@@ -81,13 +98,37 @@ export class UserService {
       }
     );
   }
+/**
+    * method to fetch appId and Ekstep_env from server.
+    */
+    public getAppidEnv(): void {
+      const url = `get/envData`;
+      this.http.get(url).subscribe(res => {
+        this._appId = res['appId'];
```

Comment:
this._appId = res.appId; use this, keep consistance

Reply:
done